hardware/opentrons_hardware/hardware_control/tool_sensors.py

```diff
@@ -44,6 +44,7 @@ async def capacitive_probe(
     mover: NodeId,
     distance: float,
     speed: float,
+    sensor_id: Optional[SensorId] = None,
```

Comment:
Do we need this `Optional`, or could we do:

```suggestion
    sensor_id: SensorId = SensorId.S0,
```